woof-code/rootfs-skeleton/etc/rc.d/rc.sysinit

```diff
@@ -557,15 +574,20 @@ udevadm trigger #--action=add --subsystem-match="input" --subsystem-match="sound
 udevadm settle -t 20
 
 # -- from slackware's /etc/rc.d/rc.udev -- modded. kmod 14+
-kmod static-nodes -f tmpfiles | \
+(
+	kmod static-nodes -f tmpfiles
+	cat /usr/lib/tmpfiles.d/static-nodes-permissions.conf 2>/dev/null
+) |
 while read F1 F2 F3 F4 F5 F6 F7 F8 ETC ; do
 	case $F1 in
 		"c!"*|"b!"*|"u!"*|"p!"*) ok=1 ;;
+		"z"*) chmod $F3 $F2 2>/dev/null ;;
 		*) continue ;;
 	esac
 	[ -z "$F7" ] && continue #precaution
 	# mknod: /dev/net/tun: No such file or directory
-	[ ! -d ${F2%/*} ] && mkdir -p -m $F3 ${F2%/*}
```

Comment:
Huge bug in this line. If /dev/net/tun is missing and needs to be 666, but /dev/net is missing, the latter should be 755 and not 666. Otherwise, spot can't `ls /dev/net` and /dev/net/tun is inaccessible.